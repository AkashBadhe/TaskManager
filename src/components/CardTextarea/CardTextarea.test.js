import React from "react";
import { shallow } from "enzyme";
import CardTextarea from "./index";

describe("CardTextarea", () => {
  let cardTextarea = shallow(<CardTextarea />);
  it("Should mount `CardTextarea`", () => {
    expect(cardTextarea).toMatchSnapshot();
  });

  it("Should have `Styled(TextareaAutosize)`", () => {
    expect(cardTextarea.find("Styled(TextareaAutosize)").exists()).toBe(true);
  });
});
