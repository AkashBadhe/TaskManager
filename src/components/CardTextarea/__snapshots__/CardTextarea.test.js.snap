// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTextarea Should mount \`CardTextarea\` 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardTextarea />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "autoFocus": true,
      "onFocus": [Function],
      "useCacheForDOMMeasurements": true,
    },
    "ref": null,
    "rendered": null,
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "baseHash": 400283751,
        "componentId": "sc-AxjAm",
        "isStatic": false,
        "rules": Array [
          "
  border-radius: 4px;
  box-shadow: inset 0 0 0 2px rgba(0, 0, 0, 0.1);
  border: none;
  padding: 8px;
  margin-bottom: 10px;
  overflow: ;
  width: 100%;
  height: 100%;
  box-sizing: border-box;
  font-family: inherit;
  font-size: inherit;
  color: inherit;
  outline: none;
  resize: none;
",
        ],
        "staticRulesId": "",
      },
      "displayName": "Styled(TextareaAutosize)",
      "foldedComponentIds": Array [],
      "render": [Function],
      "shouldForwardProp": undefined,
      "styledComponentId": "sc-AxjAm",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
      Symbol(Symbol.hasInstance): [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "autoFocus": true,
        "onFocus": [Function],
        "useCacheForDOMMeasurements": true,
      },
      "ref": null,
      "rendered": null,
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "baseHash": 400283751,
          "componentId": "sc-AxjAm",
          "isStatic": false,
          "rules": Array [
            "
  border-radius: 4px;
  box-shadow: inset 0 0 0 2px rgba(0, 0, 0, 0.1);
  border: none;
  padding: 8px;
  margin-bottom: 10px;
  overflow: ;
  width: 100%;
  height: 100%;
  box-sizing: border-box;
  font-family: inherit;
  font-size: inherit;
  color: inherit;
  outline: none;
  resize: none;
",
          ],
          "staticRulesId": "",
        },
        "displayName": "Styled(TextareaAutosize)",
        "foldedComponentIds": Array [],
        "render": [Function],
        "shouldForwardProp": undefined,
        "styledComponentId": "sc-AxjAm",
        "target": [Function],
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
        Symbol(Symbol.hasInstance): [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
